docs/apis-tools/spring-zeebe-sdk/getting-started.md

```diff
@@ -4,6 +4,11 @@ title: Getting started
 description: "Leverage Camunda APIs (gRPC and REST) in your Spring Boot project."
 ---
 
+:::info Deprecated Zeebe Spring
+Starting with 8.8, the Camunda Spring Boot replaces the Zeebe Spring. The SDK will rely on the new Camunda Java client,
```

Comment:
🔧 We should be consistent with the library name, so that users understand what we are referring to:
```suggestion
Starting with 8.8, the Camunda Spring Boot SDK replaces the Spring Zeebe. The SDK will rely on the new Camunda Java client,
```

Reply:
Agreed -- or "the Camunda Spring Boot SDK replaces the Spring Zeebe SDK"